scripts/setup/config.ps1

```diff
@@ -115,6 +115,19 @@ function Remove($_path) {
     }
 }
 
+function EnsureOwnerGroup($settings) {
```

Comment:
Can we document this function with a summary of what it does similarly to the other functions in this script?

Reply:
Also best practices say to use `Verb-Noun` format like `Ensure-OwnerGroup`.

I think the name would be better as `Ensure-IncludesIisAdminApiOwners`